plugin.yml

```diff
@@ -4,7 +4,7 @@ version: 2.0.7
 description: Slapper, the NPC plugin for PocketMine-MP
 main: slapper\Main
 api: 4.0.0
-mcpe-protocol: 503
+mcpe-protocol: 527
```

Comment:
```suggestion
```

Reply:
See #42

Reply:
I still think the `mcpe-protocol` field should be removed